src/components/pages/AdminDashboard/DayComponents/DayComponents/01_Sat-Mon.js

```diff
@@ -0,0 +1,155 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import { useOktaAuth } from '@okta/okta-react';
+import { Layout, Button, Radio } from 'antd';
+
+import { date, tasks } from '../../../../../state/actions/index';
+import { setAllTasks, getChildTasks } from '../../../../../api/index';
+import {
+  setHasDrawn,
+  setHasRead,
+  setHasWritten,
+} from '../../../../../state/actions/taskActions';
+
+const { Header, Content, Footer } = Layout;
+
+const SatMon = ({ setDate }) => {
+  const { authState } = useOktaAuth();
+  const { push } = useHistory();
+
+  const adminDash = () => {
+    push('/admin');
+  };
+
+  const gameStageUrl = '/child/mission-control';
+
+  const findDayOfWeekReference = 1;
+
+  // const handleSim = () => {
+  //   push(`${gameStageUrl}`);
+  // };
+
+  const findNextDayOfWeek = selectedDay => {
+    let date = new Date();
+    let resultDate = new Date(date.getTime());
+    resultDate.setDate(
+      //.setDate is a JS date function NOT the setDate() action used by Redux
+      date.getDate() + ((7 + selectedDay - date.getDay()) % 7)
+    );
+    return resultDate;
+  };
+
+  useEffect(() => {
+    setDate(findNextDayOfWeek(findDayOfWeekReference));
+  }, [findDayOfWeekReference]);
+
+  const handleGetChildTasks = e => {
+    getChildTasks(authState, tasks.child_id, tasks.story_id);
+  };
+
+  const handleAllTasks = e => {
+    setAllTasks(
+      authState,
+      tasks.submissionID,
+      tasks.hasRead,
+      tasks.hasWritten,
+      tasks.hasDrawn
+    );
+  };
+
+  const onSimulate = e => {
+    if (handleAllTasks() !== []) {
+      handleGetChildTasks();
+    }
+  };
+
+  const completeOneTask = e => {
```

Comment:
One idea I'm thinking we may consider is that instead of directly manipulating the state like this, we have the application search the database to view the user's submissions and set the state accordingly, so that we can make sure the database queries are working correctly.